### ISSUE TITLE
blog-d2l-video

### ISSUE THREAD

**Issue by qixinbo** [qixinbo/image-container#25]
![bp](https://user-images.githubusercontent.com/6218739/125578078-27b23a05-77f6-48a6-9e67-cf93a88d15bb.png)

**Comment by qixinbo** [comment]
![chain](https://user-images.githubusercontent.com/6218739/125578224-ee57736e-8f8d-4860-a206-5c843996a1ea.png)

**Comment by qixinbo** [comment]
![softmax](https://user-images.githubusercontent.com/6218739/125717907-efea76d0-69f5-4bda-8766-597ed23def97.png)

**Comment by qixinbo** [comment]
![crossentropy](https://user-images.githubusercontent.com/6218739/125719060-36c12bc0-724e-4d33-8765-717e4fd2328f.png)

**Comment by qixinbo** [comment]
![cnn](https://user-images.githubusercontent.com/6218739/127293555-822ddff0-48d7-488b-b1dd-7530386df069.gif)

**Comment by qixinbo** [comment]
![cnn-size](https://user-images.githubusercontent.com/6218739/127438125-4ec2e8fa-876b-4447-ac60-e005e8d9ed8a.png)

**Comment by qixinbo** [comment]
![lenet](https://user-images.githubusercontent.com/6218739/127605212-9e45557f-be04-4870-b896-791b2886d891.png)

**Comment by qixinbo** [comment]
![more-layer](https://user-images.githubusercontent.com/6218739/127817392-62cb2a30-6c0b-4429-ad19-0da9bf5749ae.png)

**Comment by qixinbo** [comment]
![resnet-block](https://user-images.githubusercontent.com/6218739/127819013-9637a861-d32e-4c0e-beef-1d676fd1e402.png)

**Comment by qixinbo** [comment]
![anchor](https://user-images.githubusercontent.com/6218739/128323707-6114e761-d06b-4c71-88d3-b95c3dba103f.png)
